workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/pages/TicketsDashboard.tsx

```diff
@@ -0,0 +1,188 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayLayout from '@clayui/layout';
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import {DndContext} from '@dnd-kit/core';
+import React, {useMemo, useState} from 'react';
+import {QueryClient, useMutation, useQueryClient} from 'react-query';
+
+import StatusColumn from '../components/StatusColumn';
+import {useTickets} from '../hooks/useTickets';
+import {Liferay} from '../services/liferay';
+import {updateTicketStatus} from '../services/tickets';
+import {Filter, ScreenType, Ticket} from '../types';
+
+type RelatedTicketsMap = {
+	[key: string]: Ticket[];
+};
+
+const DRAG_RESULT = {
+	NO_CHANGE: 'NO_CHANGE',
+	STATUS_CHANGED: 'STATUS_CHANGED',
+};
+
+const ALLOWED_DASHBOARD_STATUSES = [
+	'Open',
+	'In Progress',
+	'Answered',
+	'Closed',
+];
+
+const INITIAL_FILTER_STATE: Filter = {
+	field: '',
+	label: '',
+	value: '',
+};
+
+const DASHBOARD_STYLE_WHILE_LOADING: React.CSSProperties = {
+	opacity: 0.5,
+	zIndex: 1,
+};
+
+const PAGE_LOADING_INDICATOR_STYLE: React.CSSProperties = {
+	opacity: 1,
+	zIndex: 2,
+};
+
+const TicketsDashboard: React.FC<{screenType?: ScreenType}> = ({
+	screenType,
+}) => {
+	const queryClient: QueryClient = useQueryClient();
+
+	const [isLoading, setIsLoading] = useState(false);
+
+	const {rows: tickets} = useTickets({
+		debouncedPage: 0,
+		debouncedSearch: '',
```

Comment:
We shouldn't be renaming the parameters for useTickets be debounced. useTickets doesn't care if the page or search is debounced, it just wants a page and a search.